src/lib/UI/Config/Provider/ContentTypes.php

```diff
@@ -27,14 +27,15 @@ public function __construct(ContentTypeService $contentTypeService)
      */
     public function getConfig()
     {
-        $contentTypeGroups = [];
+        $contentTypeGroups = ['_types' => []];
 
         foreach ($this->contentTypeService->loadContentTypeGroups() as $contentTypeGroup) {
             foreach ($this->contentTypeService->loadContentTypes($contentTypeGroup) as $contentType) {
                 $contentTypeGroups[$contentTypeGroup->identifier][] = [
                     'identifier' => $contentType->identifier,
                     'name' => $contentType->getName(),
                 ];
+                $contentTypeGroups['_types'][$contentType->identifier] = $contentType->getName();
```

Comment:
Note: Not strictly needed, it could be extracted on JS side instead if we wanted to, depends on where you want to take this later.